src/staking-v3/components/dapp/DappStatistics.vue

```diff
@@ -1,83 +1,57 @@
-          </span>
-        </div>
-      </div>
-      <!-- <div class="separator--statistics" />
-      <div class="row--statistics">
-        <div>
-          <span class="text--statistics-title">
-            {{ $t('dappStaking.dappPage.totalStaker') }}
-          </span>
-        </div>
-        <div>
-          <span class="text--statistics-value">
-            {{ $n(dapp.stakerInfo.stakersCount) }}
-          </span>
-        </div>
-      </div> -->
-      <div class="separator--statistics" />
-      <div class="row--statistics">
-        <div>
-          <span class="text--statistics-title">
-            {{ $t('stakingV3.tier') }}
-          </span>
-        </div>
-        <div>
-          <span class="text--statistics-value">
-            {{ getDappTier(dapp.chain.id) ?? '--' }}
-          </span>
-        </div>
-      </div>
-      <!-- <div class="separator--statistics" />
-      <div class="row--statistics">
-        <div>
-          <span class="text--statistics-title">
-            {{ $t('dappStaking.dappPage.v3.totalEarned') }}
-          </span>
-        </div>
-        <div>
-          <span class="text--statistics-value">
-            {{ $t('amountToken', { amount: 10, token: nativeTokenSymbol }) }}
-          </span>
-        </div>
-      </div> -->
+    <div class="row--data">
+      <button class="button--vote-stake" @click="navigateToVote(dapp.extended?.address)">
+        {{ $t('stakingV3.dapp.voteAndStake') }}
+      </button>
+
+      <kpi-card :title="$t('dappStaking.dappPage.totalStaked')">
+        <token-balance-native :balance="dapp.chain.totalStake?.toString() || '0'" />
+      </kpi-card>
+
+      <!-- <kpi-card :title="$t('dappStaking.dappPage.totalStaker')">
+        <span>{{ $n(dapp.stakerInfo.stakersCount) }}</span>
+      </kpi-card> -->
+
+      <kpi-card :title="$t('stakingV3.currentTier')">
+        <span>{{ getDappTier(dapp.chain.id) ?? '--' }}</span>
+      </kpi-card>
+
+      <!-- <kpi-card :title="$t('stakingV3.totalEarned')">
+        <span>{{ $t('amountToken', { amount: 10, token: nativeTokenSymbol }) }}</span>
+      </kpi-card> -->
     </div>
   </div>
 </template>
 <script lang="ts">
 import TokenBalanceNative from 'src/components/common/TokenBalanceNative.vue';
-import { useDappStaking } from 'src/staking-v3/hooks';
+import { useDappStaking, useDappStakingNavigation } from 'src/staking-v3/hooks';
 import { CombinedDappInfo } from 'src/staking-v3/logic';
 import { defineComponent, PropType } from 'vue';
+import KpiCard from '../KpiCard.vue';
+
 export default defineComponent({
   components: {
     TokenBalanceNative,
+    KpiCard,
   },
   props: {
     dapp: {
       type: Object as PropType<CombinedDappInfo>,
       required: true,
     },
   },
-  setup() {
+  setup(props) {
     const { getDappTier } = useDappStaking();
+    const { navigateToVote } = useDappStakingNavigation();
+
+    console.log('dapp', props.dapp);
```

Comment:
Delete if this is for debugging purpose. I plan to clear all  console outputs in dApp staking expect a places where we are logging subsciption events, e.g .protocol state changed, ledger changed...

Reply:
@bobo-k2 I've deleted it in [8947a54](https://github.com/AstarNetwork/astar-apps/pull/1078/commits/8947a540b381f8e47297d76349ba06c603d443a6)